### PR TITLE
docs: add API documentation with swagger

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ services:
       - "traefik.backend=gsuite-wrapper"
       - "traefik.port=8084"
       - "traefik.frontend.rule=Path:/api/gsuite-wrapper/healthcheck;PathPrefixStrip:/api/gsuite-wrapper"
+      - "traefik.frontend.rule=Path:/api/gsuite-wrapper/api-docs.json;PathPrefixStrip:/api/gsuite-wrapper"
```
Comment (serge1peshcoff) [inline docker/docker-compose.yml:28]: won't that break the healthcheck exposing to public? as the traefik.frontend.rule is overridden

**Review by serge1peshcoff — COMMENTED** [review]
lgtm except for one comment

File: docker/docker-compose.yml (hunk 1 above)
Comment (linuxbandit) [reply docker/docker-compose.yml:28]: no, it is a different rule for what i remember (i will re-test with more attention in my setup)

Comment (linuxbandit) [reply docker/docker-compose.yml]: good catch

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**